app/javascript/layouts/Layout.tsx

```diff
@@ -31,12 +34,33 @@ const Layout = (props: LayoutProps) => {
 
   const currentPath = window.location.pathname
 
+  let notice = null
+  if (process.env.SHOW_RESEARCH_BANNER) {
+    notice = (
+      <Markdown>
+        {t("nav.researchFeedback", { researchUrl: process.env.RESEARCH_FORM_URL })}
+      </Markdown>
+    )
+  }
+  let topAlert = null
+  if (process.env.TOP_MESSAGE) {
+    topAlert = (
+      <AlertBox
+        type={(process.env.TOP_MESSAGE_TYPE as AlertTypes) || "alert"}
+        inverted={process.env.TOP_MESSAGE_INVERTED === "true" || false}
```

Comment:
```suggestion
        inverted={process.env.TOP_MESSAGE_INVERTED === "true"}
```

nit: I think `|| false` is redundant here